components/postComponents/PostHeader.tsx

```diff
@@ -27,10 +27,10 @@ export default function PostHeader({
   const [likes, setLikes] = useState(item.likes);
   const [replies, setReplies] = useState(item.replies);
   const likeDebounce = useRef(false);
-  let incrementLikeSubscription,
-    decrementLikeSubscription,
-    incrementReplySubscription,
-    decrementReplySubscription;
+  let incrementLikeSubscription: { unsubscribe: () => void; },
```

Comment:
Thanks for adding types, though the new components are more important to dedicate time to.